frontend/config/dev.env.js

```diff
@@ -3,5 +3,7 @@ const merge = require('webpack-merge')
 const prodEnv = require('./prod.env')
 
 module.exports = merge(prodEnv, {
-  NODE_ENV: '"development"'
+  NODE_ENV: '"development"',
+  MAPBOX_TOKEN: '"'+process.env.MAPBOX_TOKEN+'"',
```

Comment:
Why we have such '"'+process.env.MAPBOX_TOKEN+'"' here?

Reply:
@nemoipaha variables from .env not available in frontend code, but available here

Reply:
@aLLeXUs it is ok, i meant why we have such `'"'` symbols in front and not just 
`MAPBOX_TOKEN: process.env.MAPBOX_TOKEN`

Reply:
@nemoipaha variable without this symbols will be object, not string, but i can move them to getter

Reply:
It is ok for now )